OnlineFlightSearchAPI/Repositories/FlightRepository.cs

```diff
@@ -1,45 +1,16 @@
-﻿using OnlineFlightSearchAPI.FlightServices;
-using OnlineFlightSearchAPI.Models;
-using System;
+﻿using System;
 using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations;
 using System.Linq;
+using OnlineFlightSearchAPI.FlightServices;
+using OnlineFlightSearchAPI.Models;
 
 namespace OnlineFlightSearchAPI.Repositories.FlightRepository
 {
     public class FlightRepository : IFlightRepository
     {
-        private readonly IAirportServices _airportService;
-
         public List<FlightDetail> flightDetails { get; set; } = new List<FlightDetail>();
```

Comment:
can be private?
Better naming?

Reply:
Modifying this a private will change the application architecture, since we need this for initializing repository. But it is abstracted in IFlightRepository